src/main/java/uk/gov/hmcts/juror/api/moj/report/AbstractStandardReport.java

```diff
@@ -20,6 +20,11 @@ public AbstractStandardReport(PoolRequestRepository poolRequestRepository, Entit
         super(poolRequestRepository, from, dataType);
     }
 
+    public AbstractStandardReport(EntityPath<?> from,
+                                  DataType... dataType) {
```

Comment:
DataType should be IDataType in abstracts (With exception of group by)

Reply:
removed this definition as it was already defined in code above